packages/front-end/components/Experiment/AnalysisForm.tsx

```diff
@@ -90,12 +89,12 @@ const AnalysisForm: FC<{
         experiment.attributionModel ||
         orgSettings.attributionModel ||
         "firstExposure",
-      // @ts-expect-error TS(2345) If you come across this, please fix it!: Argument of type 'string | undefined' is not assig... Remove this comment to see the full error message
-      dateStarted: getValidDate(phaseObj?.dateStarted)
+      dateStarted: getValidDate(phaseObj?.dateStarted ?? "")
+        .toISOString()
+        .substr(0, 16),
```

Comment:
This is going to crash if `dateStarted` is null since we are not providing a fallback, which can result in undefined being returned, so calling `.toISOString()` would crash. That's what I'm seeing in the `getValidDate()` function.

Reply:
it shouldn't crash, due to `?? ""` which will force it to return today's date. Or am I missing something?

Reply:
Oh, I missed that the fallback will be set to today's date if it's not provided. Ok never mind then, that won't crash.